resources/ext.collections.styles/collections.less

```diff
@@ -90,4 +90,39 @@
 		}
 	}
 
+	.collection-card {
+		height: 14em;
+		border: 1px solid @grayLightest;
+		position: relative;
+	}
+
+	.collection-card-overlay {
+		position: absolute;
+		height: 6em;
+		background: rgba(0, 0, 0, 0.8);
+		padding: 1em;
+		bottom: 0;
+		left: 0;
+		right: 0;
+		span {
+			color: #FAF9F9;
+			padding: 0 0.25em;
+		}
+	}
+
+	.collection-card-title {
+		font-size: 2em;
+		padding-bottom: 0;
+		// Reset .mw-mf-special #content_wrapper styles for headers...
```

Comment:
We are doing this reset in a couple of places. Would should extract it to a mixin.